scikeras/wrappers.py

```diff
@@ -591,6 +570,28 @@ def fit(self, X, y, sample_weight=None):
             X=X, y=y, sample_weight=sample_weight, warm_start=self.warm_start
         )
 
+    def _initialized(self):
+        return hasattr(self, "n_features_in_")
```

Comment:
Is `n_features_in_` the best attribute to check? We also have `is_fitted_`. I feel like we should always check the same attribute, and if that's not going to be `is_fitted_` we should get rid of `is_fitted_`.

Reply:
I deleted `is_fitted_` and we can instead check `self._initialized()`.